### ISSUE TITLE
Change to the app's font

### ISSUE THREAD

**Issue by Xorio71** [Jays2Kings/tachiyomiJ2K#1378]
### Describe your suggested feature

Have you ever thought of making the app's font static, as in not changing based on the font used for the System?

For example, you could use a Material Design compliant font, like Google Sans Text.

### Other details

Example images:
![Uploading Screenshot_20220818-180824_TachiyomiJ2K~2.png…]()
![Screenshot_20220818-180830_TachiyomiJ2K~2](https://user-images.githubusercontent.com/101265862/185430627-c0c6a3a7-cc86-484b-a6d4-fb41ead518f3.png)
![Uploading Screenshot_20220818-180848_TachiyomiJ2K~2.png…]()

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an request for a feature in the original Tachiyomi, I am opening because multiple versions of TachiyomiJ2K have been released without this feature.
- [X] If this is an issue with an extension, or a request for an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[1.5.6](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/Jays2Kings/tachiyomiJ2K/issues/1378#issue-1343254001

@Xori71, why would this be desirable? It would be hideous for anyone with accessibility needs.

**Comment by Xori71** [comment]
> > [#1378 (comment)](https://github.com/Jays2Kings/tachiyomiJ2K/issues/1378#issue-1343254001)
> 
> @Xori71, why would this be desirable? It would be hideous for anyone with accessibility needs, and unexpectedly inconsistent for anyone else. If applications don't adhere to system preferences, why do those preferences exist...?

Fair point, but sometimes, it is desirable for an app to has its own design language, irrespective of user choice.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/Jays2Kings/tachiyomiJ2K/issues/1378#issuecomment-2030794227

@Xori71, when and why? That's totally unsubstantiated, especially considering the detriment to accessibility.

**Comment by Xori71** [comment]
> > [#1378 (comment)](https://github.com/Jays2Kings/tachiyomiJ2K/issues/1378#issuecomment-2030794227)
> 
> @Xori71, when and why? That's totally unsubstantiated, especially considering the detriment to accessibility.

Do Google apps conform to your font choices? Do most websites?

Mihon and its forks are going for a Material Design look, and Material Design has guidelines that are to be followed in order to achieve a complete look.

My post wasn’t a demand. It was a suggestion from a *design standpoint*. It makes sense for J2K to use a font that conforms to Material Design Guidelines.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/Jays2Kings/tachiyomiJ2K/issues/1378#issuecomment-2030802368

Yes, @Xori71, every website adheres to https://support.mozilla.org/en-US/kb/change-fonts-and-colors-websites-use#w_custom-fonts, and every local application that I've utilized conforms to what https://invent.kde.org/plasma/plasma-desktop/-/blob/e5e5f5ed51aadfac99bfbdf3d2db5be16a12443b/kcms/fonts/fontssettings.kcfg#L7-82 defines. What you describe is why so many must ask questions like https://www.reddit.com/r/GooglePixel/comments/6de01x/change_system_font_without_rooting/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button.

Regardless, if this were to be implemented, perhaps we could at least set https://developer.android.com/reference/android/view/View#attr_android:importantForAccessibility (per https://developer.android.com/guide/topics/ui/accessibility/principles#decorative) *if* that would make any difference.
